www/_template/guides/connecting-tools.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layouts/guide.njk
+title: The Snowpack Guide to connecting your favorite tools
+description: 'How do you use your favorite tools in Snowpack? This Guide will help you get started'
+tags: guides
+sidebarTitle: Connecting your favorite tools
+---
+
+One of the biggest questions we get in our discussion forums and Discord is how to connect tools to Snowpack. EsLint, PostCSS, SASS, and any other of the useful tools developers use to build JavaScript these days. In this Guide we'll go over how to use your favorite tools with Snowpack. Almost all work with just a few additions to a configuration file.
+
+## The three ways
+
+It's worth first going over the three ways and their pros and cons:
+
+- Snowpack plugins
+- Run scripts using Snowpack
+- Run scripts outside of Snowpack (in `package.json`)
+
+## Plugin
+
+Usually Snowpack plugins are the easiest way to use your favorite tools. Snowpack plugins are built to require no configuration, but if you do want to do configuration, they are built with that in mind too!
+
+Snowpack has a [growing plugin ecosystem](/plugins). Adding a plugin is two steps: install the Plugin using your package manager and then tell Snowpack about it by adding the name of the plugin to the Snowpack config file. Configuration options are covered in each plugin's documentation.
```

Comment:
I like that you kept this section short, but would it make sense to have a short example here for the “install“ and “add to config” steps? (feel free to say no; it would distract from the point).

Reply:
Yup, +1 for some code snippets walking you through this.

Also, now that we have `snowpack init`, you can handle the case where a user does not have a snowpack.config.js file yet.